src/System.Private.CoreLib/shared/System/Diagnostics/StackFrame.cs

```diff
@@ -47,7 +47,7 @@ public partial class StackFrame
         /// <summary>
         /// This flag is set to true when the frame represents a rethrow marker.
         /// </summary>
-        private readonly bool _isLastFrameFromForeignExceptionStackTrace;
+        private bool _isLastFrameFromForeignExceptionStackTrace;
```

Comment:
Sorry, didn't notice corert writes to this.
https://github.com/dotnet/corert/blob/227a11d6e7462a68b691e800bd0723d63fc618b2/src/System.Private.CoreLib/src/System/Diagnostics/StackFrame.CoreRT.cs#L132

Reply:
No problem. Thank you for doing these cleanups.

This whole type is logically readonly, but the fields cannot be really marked as readonly given how the construction is factored...